fog/sample-paykit/src/client.rs

```diff
@@ -534,13 +533,16 @@ impl Client {
         Ok(res.num_blocks + self.new_tx_block_attempts as u64)
     }
 
-    /// Retrieve the currently configured minimum fee from the consensus service
-    pub fn get_fee(&mut self) -> Result<u64> {
-        Ok(self
-            .consensus_service_conn
-            .fetch_block_info()?
-            .minimum_fee_or_none(&Mob::ID)
-            .unwrap_or(0))
+    /// Retrieve the current last block info structure from consensus service.
+    /// This includes fee data and last block index
+    pub fn get_last_block_info(&mut self) -> Result<BlockInfo> {
```

Comment:
i'm exposing this also because i think it might be useful for test client later, partly based on discussions with sdk team recently